libs/brain/tooltip/src/lib/brn-tooltip-trigger.directive.ts

```diff
@@ -392,15 +332,15 @@ export class BrnTooltipTriggerDirective implements OnDestroy, AfterViewInit {
 				}
 			});
 
-		if (brnDevMode && !this._ariaDescribedBy) {
+		if (brnDevMode && !this.ariaDescribedByState()) {
```

Comment:
again I don't feel like I know why there is a ariaDescribedByState variable

Reply:
I changed it and i also removed the ariaDescribedByState